src/cli/remove.rs

```diff
@@ -44,22 +54,43 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     let results = deps
         .iter()
         .map(|dep| {
-            if let Some(p) = &args.platform {
-                project
-                    .manifest
-                    .remove_target_dependency(dep, &spec_type, p)
+            if args.pypi {
+                DependencyRemovalResult::PyPiDeps(project.manifest.remove_pypi_dependency(dep))
```

Comment:
PyPi dependencies can also be target specific.

Reply:
but isn't using the ``PyPiRequirements`` where the target platform is specified there?

Reply:
I don't understand the question. 

But you can have:
```toml
[target.win-64.pypi-dependencies]
requests = "*"
```

Reply:
gotcha, so things are a bit complex then because you can specify platform in the ``PyPiRequirements`` in the extras field

Reply:
something like:
```
extras= platform_system != "Windows"
```

Reply:
I wasn't aware we support that, for this feature you will need to support the pixi's target part.

Reply:
this is the only bit missing, gonna continue with that tomorrow :)

Reply:
done